workflow/hosts/gaeac5.yaml

```
@@ -24,5 +24,7 @@ LOCALARCH: 'NO'
 ATARDIR: '${NOSCRUB}/archive_rotdir/${PSLOT}'
 MAKE_NSSTBUFR: 'NO'
 MAKE_ACFTBUFR: 'NO'
-SUPPORTED_RESOLUTIONS: ['C384', 'C192', 'C96', 'C48']
+DO_FETCH_HPSS: 'NO'
+SUPPORTED_RESOLUTIONS: ['C1152', 'C768', 'C384', 'C192', 'C96', 'C48']
```

I think this got mangled in the merge.  This should be

```suggestion
SUPPORTED_RESOLUTIONS: ['C384', 'C192', 'C96', 'C48']
```

---

I think the higher resolutions need to stay in the supported list.

---

Corrected that and repushed.